devnet-sdk/system/chain.go

```diff
@@ -106,13 +107,13 @@ func (m *clientManager) GethClient(rpcURL string) (*ethclient.Client, error) {
 type chain struct {
 	id          string
 	rpcUrl      string
-	users       map[string]Wallet
+	users       WalletMap
```

Comment:
It might be more intuitive to change the field name to `wallets`.

Reply:
Fixed in 2895f546